shared/src/business/useCases/getReconciliationReportInteractor.ts

```diff
@@ -30,35 +31,57 @@ const isValidDate = dateString => {
  */
 export const getReconciliationReportInteractor = async (
   applicationContext: IApplicationContext,
-  { reconciliationDate }: { reconciliationDate: string },
+  {
+    end: timeEnd,
+    reconciliationDate,
+    start: timeStart,
+  }: {
+    reconciliationDate: string;
+    end?: string;
+    start?: string;
+  },
 ) => {
   const authorizedUser = applicationContext.getCurrentUser();
-
   if (!isAuthorized(authorizedUser, ROLE_PERMISSIONS.SERVICE_SUMMARY_REPORT)) {
     throw new UnauthorizedError('Unauthorized');
   }
 
-  if (reconciliationDate === 'today') {
-    reconciliationDate = formatNow(FORMATS.YYYYMMDD);
-  } else {
-    const dateInputValid = isValidDate(reconciliationDate);
-    if (!dateInputValid) {
-      throw new Error(
-        'Date must be formatted as YYYY-MM-DD and not later than today',
-      );
-    }
+  const effectiveTimeStart = isValidTime(timeStart!) ? timeStart : '00:00';
+  const effectiveTimeEnd = isValidTime(timeEnd!)
+    ? `${timeEnd}:59.999`
+    : '23:59:59.999';
+
+  const effectiveReconciliationDate =
+    reconciliationDate == 'today'
+      ? formatNow(FORMATS.YYYYMMDD)
+      : reconciliationDate;
+
+  if (!isValidReconciliationDate(effectiveReconciliationDate)) {
+    throw new InvalidRequest(
+      'Must be valid reconciliation date and not later than today',
+    );
   }
 
-  const [year, month, day] = reconciliationDate.split('-');
-  const reconciliationDateStart = createStartOfDayISO({ day, month, year });
-  const reconciliationDateEnd = createEndOfDayISO({ day, month, year });
+  //convert to full iso-8601 time stamps in utc timezone
+  const { end: isoEnd, start: isoStart } = normalizeIsoDateRange(
+    `${effectiveReconciliationDate}T${effectiveTimeStart}`,
+    `${effectiveReconciliationDate}T${effectiveTimeEnd}`,
+  );
+
+  const hours = calculateDifferenceInHours(isoEnd, isoStart);
+  if (hours > MAX_TIMESPAN_HOURS) {
+    throw new InvalidRequest(
+      `Range must not exceed ${MAX_TIMESPAN_HOURS} hours`,
+    );
+  }
```

Comment:
Just curious -

I think I recall you saying that the start and end times submitted by the end user are in Eastern time. If the start and end are a full day, but that day happens to span the "fall back" time change, won't this error be thrown?